deface.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.summary = "Deface is a library that allows you to customize ERB, Haml and Slim views in Rails"
 
-  s.add_dependency('nokogiri', '~> 1.6')
+  s.add_dependency('nokogiri', '>= 1.6')
```

Comment:
@klevo 
```
gem 'nokogiri', '~> 1.6', '>= 1.6.0'
```
maybe?

Reply:
It really depends on what we want to achieve here. If we want security to be the priority and want people to force to upgrade nokogiri in their project, then it should be:

```
s.add_dependency('nokogiri', '~> 1.8.1')
```

If compatibility is the primary concern then the original or your solution seems to work. It really depends on the conventions of this particular repository, so I'd leave this choice up to you.